pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -634,6 +686,10 @@ func (p *criStatsProvider) getContainerUsageNanoCores(stats *runtimeapi.Containe
 		return nil
 	}
 
+	if stats.Cpu != nil && stats.Cpu.UsageNanoCores != nil {
```

Comment:
just to understand, reason we don't need to acquire mutex etc, is because we are not using the "cached" `usageNanoCores` and instead using one directly supplied from the stats message?

Reply:
is there any reason for us to cache this `UsageNanoCores` in CRI stats world or is it unnecessary now?

Reply:
I don't see much need to deal with locking if we're going to bypass the cache now